dxfReader.py

```diff
@@ -258,22 +260,24 @@ def handleBlock(block, infile):
 """
 
 def start(cargo):
-	"""Expects the infile as cargo, initializes the cargo."""
+	"""Expects (infile, acadVersion) as cargo, initializes the drawing."""
```

Comment:
Here I would explicitly change the function definition to, for ex `def start(cargo,acadVersion=XXX)` or even without the default value for acadVersion. Otherwise other people using this code (some other modules like openSCAD use it) might be unaware of the change, and this will give errors that will be hard to trace back. Better change the function arguments so people get a clean error that is easy to trace and fix